src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/desktop/Promoting.java

```diff
@@ -1,13 +1,25 @@
 package com.wikia.webdriver.elements.mercury.components.discussions.desktop;
 
+import com.google.common.base.Predicate;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.google.common.base.Predicate'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)